installation/scripts/testing.sh

```diff
@@ -110,6 +110,12 @@ function cleanupHelmTestPods() {
 
     log "\nCleaning up helm test pods" nc bold
     kubectl delete pod -n ${namespace} -l helm-chart-test=true
+    deleteErr=$?
+    if [ ${deleteErr} -ne 0 ]
+    then
+      log "FAILED cleaning test pods.\n" red
```

Comment:
Two things, first, I cannot see any other case where we are logging msg with `FAILED` when `kc delete` will fail then this command should already print some message, right?
But if you want to have it here then you should adjust the log in line 119, `End of cleaning test pods.\n` it should be sth like `Success cleaning test pods.\n`

Reply:
Done